focus-blocked/script.js

```diff
@@ -24,7 +24,7 @@ const focus_tip_old_url = `<div class='notice-wrapper'>
                     ${
                       block_type
                         ? ''
-                        : `<a href=${current_url}>Click here to re-open the original URL ${
+                        : `<a href=${`https://${current_url}`}>Click here to re-open the original URL ${
```

Comment:
I don't think this is the right approach. https:// is already there in some cases. You need to process old_url to add https:// if it is not set

Reply:
![image](https://user-images.githubusercontent.com/11420548/224526129-a67178f9-d5a6-4404-a179-cf810f850016.png)